cypress/fixtures/successful_university_login.json

```diff
@@ -1,9 +1,10 @@
 {
   "data": {
-    "id": 2,
+    "id": 1,
+    "name": "Harvard",
     "email": "harvard@mail.com",
     "uid": "harvard@mail.com",
-    "role": "university",
```

Comment:
This should be kept to lowercase "university" because that's how the data is stored in the backend.  We've got "university", "research_group", and "reader".  I don't think rails can tell the difference.

Reply:
fixed! now all is in lower case to match API